how-to/software/automatic-updates.md

```diff
@@ -353,6 +353,8 @@ The feature is disabled if the number of days is set to 0.
 
 The `Unattended-Upgrade::Postpone-Wait-Time` configuration option controls the amount of time (in seconds) that a user has available to send a postpone request after being prompted. If no postpone request is received within the specified time, the updates will start being applied as normal.
 
+The system administrator can restrict access to the postpone request by defining Polkit rules for the `com.ubuntu.UnattendedUpgrade.Pending.Postpone` action. By default, access is granted to users of an active session.
```

Comment:
We don't have any documentation in the server docs (currently) about Polkit - do we have anything on the Desktop side that exists that we can point to?

Reply:
No, but there's https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html which we could link to?

It has an "AUTHORIZATION RULES" section which is what I mention here.

Reply:
It's also available as a manpage: https://manpages.ubuntu.com/manpages/plucky/man8/polkit.8.html

Reply:
Both look like nice references - could you add them both? Then I'll be happy to merge :)

Reply:
Well it's the same content in both references, except freedesktop.org has images while manpages.ubuntu.com has ASCII art. I don't think referencing both makes sense, then which one would you point to?

Reply:
Usually the way we do this (we do it quite often) is if there's a web version we provide a hyperlink to the external URL, and if there's a manpage we show the man command.

Reply:
I think that the freedesktop link has more value than the man page because the diagram is more readable and it features two screenshots, too. If we want to add the man page as well, I don't mind, but it only duplicates the information in a less readable format.